doc/man3/SSL_CTX_set_cert_comp_cb.pod

```diff
@@ -0,0 +1,179 @@
+=pod
+
+=head1 NAME
+
+SSL_CTX_set_cert_comp_preference,
+SSL_set_cert_comp_preference,
+OSSL_cert_comp_fn,
+SSL_CTX_set_cert_comp_cb,
+SSL_set_cert_comp_cb,
+SSL_CTX_get_cert_to_compress,
+SSL_CTX_set_compressed_cert,
+SSL_get_cert_to_compress,
+SSL_set_compressed_cert,
+SSL_CTX_compress_cert - Certificate compression functions
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ int SSL_CTX_set_cert_comp_preference(SSL_CTX *ctx, uint16_t *algs, size_t len);
+ int SSL_set_cert_comp_preference(SSL *ssl, uint16_t *algs, size_t len);
+
+ typedef int (*OSSL_cert_comp_fn)(SSL *ssl, uint16_t alg, void *arg);
+ int SSL_CTX_set_cert_comp_cb(SSL_CTX *ctx, OSSL_cert_comp_fn cert_comp_cb, void *arg);
+ int SSL_set_cert_comp_cb(SSL *ssl, OSSL_cert_comp_fn cert_comp_cb, void *arg);
+
+ unsigned char *SSL_CTX_get_cert_to_compress(SSL *ssl, size_t *length);
+ int SSL_CTX_set_compressed_cert(SSL *ssl, uint16_t algorithm, unsigned char *comp_data,
+                              size_t comp_length, size_t orig_length);
+ unsigned char *SSL_get_cert_to_compress(SSL *ssl, size_t *length);
+ int SSL_set_compressed_cert(SSL *ssl, uint16_t algorithm, unsigned char *comp_data,
+                              size_t comp_length, size_t orig_length);
+
+ int SSL_CTX_compress_cert(SSL_CTX *ctx, uint16_t alg);
+
+=head1 DESCRIPTION
+
+These functions control the certificate compression feature. Certificate
+compression is only available for TLSv1.3 as defined in RFC8879.
+
+SSL_CTX_set_cert_comp_preference() and SSL_set_cert_comp_preference() are used
+to specify the preferred compression algorithms. Only those algorithms enabled
+in the library will be accepted, unknown algorithms are ignored. The B<algs>
```

Comment:
BOTH!

Reply:
Specifically, this is specifying that this API call will only accept algorithms that are enabled in the library. A later paragraph explains how the preference lists are used.
```
The above is also the default preference order. If a preference order is not
specified, then the default preference order is sent to the peer and the
received peer's preference order will be used when compressing a certificate.
Otherwise, the configured preference order is sent to the peer and is used
when compressing the certificate.